sdk/ai/azure-ai-agents/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@
 - Added a sample showing auto function call for a synchronous client, [`sample_agents_auto_function_call.py`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-agents/samples/agents_tools/sample_agents_auto_function_call.py)
 - Added a sample showing auto function call for an asynchronous client, [`sample_agents_auto_function_call_async.py`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ai/azure-ai-agents/samples/agents_async/sample_agents_auto_function_call_async.py).
 
+### Bugs Fixed
+
+- `_AgentsClientOperationsMixin` is now private.
```

Comment:
I don't think we need to list this as a bugfix. No-one should have used this type. And it is unlikely that they accidentally did.